ep_jedi.py

```diff
@@ -14,7 +14,16 @@
 def get_completions(source, row, col, script_path=""):
     return []
 
-
+def real(source, row, col, script_path=""):
+    completions = []
```

Comment:
You might need to import `jedi` again in this function? Does this work in Linux 🤔

Reply:
yes, it do work

Reply:
infact my env is Ubuntu 20.04 ; )